### PR TITLE
intil commit with ol1 dding number

### PR DESCRIPTION
**Pull request by aneeshs005**

### CODE REVIEW

File: Chapter 1/Solution1.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+namespace Problem1
+{
+    public class Solution1
+        {
+            public int Add (int a,int b)
+            {
+            return a+b;
```
Comment (rohithkrajan) [inline Chapter 1/Solution1.cs]: keep the indentation correctly

File: Chapter 1/Solution2.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+namespace Problem2
+{
+    public class Solution2
+    {
+        public int AddThree(int a,int b, int c)
+        {
+            return a+b+c;
+        }
+    }
+}
```
Comment (rohithkrajan) [inline Chapter 1/Solution2.cs]: add the new line

File: README.md [hunk 3]
```diff
@@ -1 +1,2 @@
-# C-Sharp-Curriculum
\ No newline at end of file
+# C-Sharp-Curriculum
+ contis step by step guide for leaning c#
```
Comment (rohithkrajan) [inline README.md:2]: add a proper description

File: Chapter 1/Tests/Solution2Test.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        } 
+        [Test]
+        public void Test_Add_Three_Numbers_Different_Sign()
+        {
+            Solution2 solution=new Solution2();
+            int Total=solution.AddThree(5,-10,15);
+            Assert.AreEqual(10,Total);
+        } 
+    }
+}
```
Comment (rohithkrajan) [inline Chapter 1/Tests/Solution2Test.cs]: newline here

File: Chapter 1/Tests/Solution1Test.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        }
+        [Test]
+        public void Test_Add_Two_Negative_Numbers()
+        {
+            Solution1 solution=new Solution1();
+            int Total=solution.Add(-5,-5);
+            Assert.AreEqual(-10,Total);
+        }
+    }
+}
```
Comment (rohithkrajan) [inline Chapter 1/Tests/Solution1Test.cs]: newline here

File: Chapter 1/Solution3.cs [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+        {
+                if(a>5)
+                {
+                    return true;
+                }
+                else 
+                return false;
+        }
+    }
+}
```
Comment (rohithkrajan) [inline Chapter 1/Solution3.cs]: newline here

File: Chapter 1/Solution3.cs [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    public class Solution3
+    {
+        public bool GreaterThan5(int a)
+        {
+                if(a>5)
+                {
+                    return true;
+                }
+                else 
+                return false;
```
Comment (rohithkrajan) [inline Chapter 1/Solution3.cs]: indent the return statement.Also add the paratheses even for single statement.

**Review by rohithkrajan — CHANGES_REQUESTED** [review]

File: Chapter 1/Solution1.cs (hunk 1 above)
Comment (aneeshs005) [reply Chapter 1/Solution1.cs]: Corrected indentation.

File: Chapter 1/Solution2.cs (hunk 2 above)
Comment (aneeshs005) [reply Chapter 1/Solution2.cs]: Added new line

File: README.md (hunk 3 above)
Comment (aneeshs005) [reply README.md:2]: Updated description

File: Chapter 1/Tests/Solution2Test.cs (hunk 4 above)
Comment (aneeshs005) [reply Chapter 1/Tests/Solution2Test.cs]: Added new line

File: Chapter 1/Tests/Solution1Test.cs (hunk 5 above)
Comment (aneeshs005) [reply Chapter 1/Tests/Solution1Test.cs]: Added new line

File: Chapter 1/Solution3.cs (hunk 6 above)
Comment (aneeshs005) [reply Chapter 1/Solution3.cs]: Added new line

File: Chapter 1/Solution3.cs (hunk 7 above)
Comment (aneeshs005) [reply Chapter 1/Solution3.cs]: Corrected indentations and added paratheses

**Review by rohithkrajan — APPROVED** [review]

**Pull request merged**